README.rst

```diff
@@ -24,6 +24,26 @@ Development
       :target: https://coveralls.io/r/networkx/networkx?branch=master
 
 
+Download
+--------
+
+Get the latest version of NetworkX from
+https://pypi.python.org/pypi/networkx/
+
+::
+
+    $ pip install networkx
+
+To get the git version do
+
+::
+
+    $ git clone git://github.com/networkx/networkx.git
```

Comment:
If it's a `download` section, then this line is fine but `pip install` doesn't fit here. Rather make it `Installation` (which is more general) and explain the installation from source.